packages/dcos-check-runner/extra/dcos-checks-poststart.service

```diff
@@ -1,9 +1,15 @@
 [Unit]
 Description=DC/OS Poststart Checks: Run node-poststart checks
+
 [Service]
+Restart=always
```

Comment:
`Restart=on-failure` in conjunction with `RestartSec=1` and `StartLimitBurst=5` will do what we need here in that it runs the poststart checks and when they fail restarts them 5 times at one-second intervals.

Reply:
It's unclear to me why we want that behavior. If the intention is to quickly re-run checks in the event of a flaky check, our effort is probably better spent on fixing the flake. If we want to run checks more frequently, we should just reduce the interval between check runs. This unit is often an early stop when debugging a cluster, so it's valuable that it can be described simply: it runs checks once every minute.